deploy/olm/rhel-changes.yaml

```diff
@@ -15,7 +15,7 @@ metadata.annotations.alm-examples: |-
       "spec": {
         "namespace": "kube-system",
         "secretRefName": "storageos-api",
-        "secretRefNamespace": "default",
+        "secretRefNamespace": "openshift-operators",
```

Comment:
Editing this file by hand is fine but the changes will be overwritten by release-gen.sh script when it's run the next, if there's anything that doesn't match the templates in the script. Same for the two storageos.clusterserviceversion.yaml files.
I believe it's not documented anywhere how to update the CSVs. The changes should be expressed in release-gen.sh file only and when the release-gen.sh script is run before the next release, it'll update this file and the CSV files.
No harm in manually editing them for now.